### DISCUSSION TITLE
tail-based sampling support

### DISCUSSION THREAD

**Discussion by Sidhartha-Biswal** [jaegertracing/jaeger#5093] [Q&A] [answered]
Hi,

Does Jaeger support tail-based sampling?

If it supports then is there any CLI flag to pass the tail_sampling_config.yaml file.

**Comment by 88vishnu** [comment]
As per my exploration it seems  OpenTelemetry Collector support Tail-based Sampling but Jeger-collector does not support it.
Also it seems some discussion was ongoing in 2020 but no exact update available for the same.
see https://github.com/jaegertracing/jaeger/issues/425

And all the possible Sampling with Jaeger-Collector is available here https://www.jaegertracing.io/docs/1.53/sampling/#remote-sampling

**Comment by yurishkuro** [comment, marked as answer]
Tail based sampling today is [supported by opentelemetry collector](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md). Once Jaeger-v2 is released the same plugin would be usable directly in Jaeger.

**Reply by Sidhartha-Biswal** [reply]
Hi @yurishkuro,

When will Jaeger-v2 be released? As part of which Jaeger version will this be released?

**Reply by 88vishnu** [reply]
https://github.com/jaegertracing/jaeger/issues/4843#issuecomment-1888337671

**Reply by Sidhartha-Biswal** [reply]
Hi @88vishnu,

thanks
